test/api/peers.js

```diff
@@ -30,6 +33,7 @@ before(function (done) {
 });
 
 after(function () {
+	wsServer.options.port = originalWsServerPort;
 	wsServer.stop();
```

Comment:
We set the port to `wsServerPort` variable before starting websocket server, and before stoping, we set the to `originalWsServerPort`. Is `wsServer.stop()` independent of the port number?

Reply:
yes, but for clarity it should be done after, thanks